on-the-web/index.html

```diff
@@ -52,8 +52,12 @@ <h3 class="social-media-feature-section-headline"><i class="fa fa-twitter" aria-
             </div>
             <div class="marketing-site-content-section-block">
                 <h3 class="marketing-site-content-section-block-header">OME Team Meeting Minutes</h3>
```

Comment:
```suggestion
                <h3 class="marketing-site-content-section-block-header">OME Team Meeting</h3>
```

Reply:
I'll push a commit for "OME Community Meeting"